# define "running instances" of plugins, there can be more than one for each
#
WaypointPlanner:
  address: "iroc_fleet_manager/WaypointPlanner" 
  name_space: "waypoint_planner" 
CoveragePlanner:
  address: "iroc_fleet_manager/CoveragePlanner" 
  name_space: "coverage_planner" 
